woocommerce/woocommerce-ios
# JCP: Install Jetpack intro screen

Part of #5365 

# Description
This PR adds the initial screen of the Install Jetpack flow. This screen is currently presented when tapping on Install Jetpack action button on the Jetpack Benefits screen.

@adamzelinski - can you please check out if it's okay to keep the accent colors for buttons on dark mode? On the design both "Cancel" and "Get Started" buttons are white, but those do not match button styles on the app.

# Screenshots
| Light mode | Dark mode |
| ----- | ----- |
| <img src="https://user-images.githubusercontent.com/5533851/142553275-ef956c30-a74a-4a29-822c-f044ee353d98.png" width=320 /> | <img src="https://user-images.githubusercontent.com/5533851/142553314-2bbd37f6-9aea-4bf9-a28a-215de6e90641.png" width=320 />

# Testing
1. Make sure that `.jetpackConnectionPackageSupport` feature flag is on.
2. Tap on Jetpack Benefits banner, which will redirect you to Jetpack Benefits screen.
3. Tap on Install Jetpack button on Jetpack Benefits screen, notice that you are redirected to a new screen for introduction to Install Jetpack flow.

Update release notes:
